chapters/ch52.md

```diff
@@ -29,11 +29,18 @@
 
 > One of the most exciting developments in end-to-end deep learning is that it is letting us directly learn ​*y​* that are much more complex than a number. In the image-captioning example above, you can have a neural network input an image (​*x*)​ and directly output a caption (​*y​*).
 
---> _replace THIS LINE by your translation for the above line_
+> |        Bài toán            |           X             |           Y        |
+> | :----------------------- | :---------------------- | :------------------|
+> | Phân loại email rác      | Email                   | email rác/ không rác(0/1) |
+> | Nhận dạng ảnh            | Ảnh                     | Nhãn số nguyên     |
+> | Dự đoán giá nhà đất      | Đặc trưng của căn nhà   | Giá theo Đô-la     |
+> | Gợi ý sản phẩm           | Đặc trưng của sản phẩm và người dùng | Chance of purchas |
```

Comment:
"Chance of purchase" -> "khả năng mua sản phẩm". Đúng ra dịch là "Xác suất mua sản phẩm" sẽ dễ hiểu hơn. Nhưng mình nghĩ cái nào cũng ok.